frontend/packages/data-portal/app/components/Breadcrumbs.tsx

```diff
@@ -44,16 +45,22 @@ function Breadcrumb({
       {text}
     </Link>
   ) : (
-    <p className={cns(className, 'font-semibold')}>{text}</p>
+    <p className={cns(className, 'font-normal')}>{text}</p>
   )
 }
 
 export function Breadcrumbs({
   variant,
   data,
+  type = "breadcrumb-dark",
+  classname,
+  activeBreadcrumbText
 }: {
   variant: 'dataset' | 'deposition' | 'run'
-  data: { id: number; title: string }
+  data: { id: number; title: string },
+  type?: string,
+  classname?: string,
```

Comment:
This feels a little bit like maybe the variant could establish some of these instead?

Reply:
E.g. the variant could setup the className, and even maybe the activateBreadCrumbText by allowing the data to have a runID or similar

Reply:
type is possibly one that makes sense to leave as you have it because then each variant can be in light or dark mode

Reply:
I added new variant to handle breadcrumb style if we are on neuroglancer page, and variant is called "neuroglancer"
I deleted classname prop as well so the style changes according to "neuroglancer" variant only
activeBreadCrumbText is only available for breadcrumb component with "neuroglancer" variant

Not sure if it is a reliable solution. Please let me know if any changes needed or you want to discuss this @seankmartin